parser_library/src/context/instruction.h

```diff
@@ -303,11 +394,14 @@ class machine_instruction
 
     const checking::machine_operand_format* m_operands;
 
+    supported_system m_system_support;
```

Comment:
any chance squeezing in the new fields without growing the overall structure size? Maybe
```
    unsigned short m_size_in_bits : 2, m_page_no : 14; // size is only 16,32,48 bits i.e. 0x10,0x20,0x30 (low nibble is always zero), PoP has less than 16k pages
    supported_system m_system_support;

    mach_format m_format;
    reladdr_transform_mask m_reladdr_mask;
    unsigned char m_optional_op_count;
    unsigned char m_operand_len;
...
```